pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "corvus_python"
```

Comment:
Can we use a hyphen here plz? In line with our discussion during the IOC Ops call the other day. It's much more common to use hyphens rather than underscores in project names. You don't need to change anything else though.